var/spack/repos/builtin/packages/py-sphinx-rtd-theme/package.py

```diff
@@ -20,10 +21,13 @@ class PySphinxRtdTheme(PythonPackage):
 
     depends_on("py-setuptools", type="build")
     depends_on("py-sphinx", when="@0.4.1:", type=("build", "run"))
-    depends_on("py-sphinx@1.6:", when="@1:", type=("build", "run"))
+    depends_on("py-sphinx@1.6:6", when="@1:", type=("build", "run"))
```

Comment:
Technically there's no upper bound for 1.0.0 but we can keep this to play it safe

Reply:
I agree